subsys/net/openthread/rpc/common/ot_rpc_common.h

```diff
@@ -10,43 +10,37 @@
 #include <openthread/dataset.h>
 
 #include <nrf_rpc_cbor.h>
-
-#define OT_RPC_TIMESTAMP_LENGTH(timestamp) \
-	sizeof((timestamp).mSeconds) + 1 + \
-	sizeof((timestamp).mTicks) + 1 + \
-	sizeof((timestamp).mAuthoritative)
-
-#define OT_RPC_SECURITY_POLICY_LENGTH(policy) \
-	sizeof((policy).mRotationTime) + 1 + 10
-
-#define OT_RPC_OPERATIONAL_COMPONENTS_LENGTH(components) \
-	sizeof((components).mIsActiveTimestampPresent) + \
-	sizeof((components).mIsPendingTimestampPresent) + \
-	sizeof((components).mIsNetworkKeyPresent) + \
-	sizeof((components).mIsNetworkNamePresent) + \
-	sizeof((components).mIsExtendedPanIdPresent) + \
-	sizeof((components).mIsMeshLocalPrefixPresent) + \
-	sizeof((components).mIsDelayPresent) + \
-	sizeof((components).mIsPanIdPresent) + \
-	sizeof((components).mIsChannelPresent) + \
-	sizeof((components).mIsPskcPresent) + \
-	sizeof((components).mIsSecurityPolicyPresent) + \
-	sizeof((components).mIsChannelMaskPresent)
-
-#define OPERATIONAL_DATASET_LENGTH(dataset) \
-	OT_RPC_TIMESTAMP_LENGTH((dataset)->mActiveTimestamp) + \
-	OT_RPC_TIMESTAMP_LENGTH((dataset)->mPendingTimestamp) + \
-	sizeof((dataset)->mNetworkKey.m8) + 1 + \
-	sizeof((dataset)->mNetworkName.m8) + 1 + \
-	sizeof((dataset)->mExtendedPanId.m8) + 1 + \
-	sizeof((dataset)->mMeshLocalPrefix.m8) + 1 + \
-	sizeof((dataset)->mDelay) + 1 + \
-	sizeof((dataset)->mPanId) + 1 + \
-	sizeof((dataset)->mChannel) + 1 + \
-	sizeof((dataset)->mPskc.m8) + 1 + \
-	OT_RPC_SECURITY_POLICY_LENGTH((dataset)->mSecurityPolicy) + \
-	sizeof((dataset)->mChannelMask) + 1 + \
-	OT_RPC_OPERATIONAL_COMPONENTS_LENGTH((dataset)->mComponents)
+#include <openthread/ip6.h>
+
+#define OT_RPC_TIMESTAMP_LENGTH(timestamp)                                                         \
```

Comment:
Could you revert this change? There is no value in this just makes macros representation cumbersome.

Reply:
fixed.